controllers/host/host_controller.go

```diff
@@ -2073,6 +2147,78 @@ func (r *HostReconciler) Reconcile(ctx context.Context, request ctrl.Request) (r
 		return common.RetrySystemNotReady, nil
 	}
 
+	if !r.CloudManager.GetHostUpdateRoutinesRunning() {
+		logHost.Info("Starting goroutines to receive host strategy updates from other controllers")
+		go func() {
```

Comment:
The two go-routines should not be looping infinitely except for a grace shutdown.

Reply:
Implemented a method to exit out of loop / goroutine in manager.go - `CloseHostRoutine()`.

Reply:
Where does it being called?

Reply:
Further, request a change of the methods name with "Routine", please use the purpose of the methods instead